libusb should be <dllmap dll="libusb" target="libusb-0.1.so.4" />, but
it's not used currently
--- njb-sharp-0.3.0/src/njb-sharp.dll.config.in.orig	2006-03-30 06:10:52.000000000 +0200
+++ njb-sharp-0.3.0/src/njb-sharp.dll.config.in	2007-07-15 14:19:27.349677840 +0200
@@ -1,6 +1,5 @@
 <configuration>
-	<dllmap dll="libnjb" target="libnjb.so.5.1.0" />
-	<dllmap dll="libusb" target="libusb-0.1.so.4.4.2" />
-	<dllmap dll="libusb" target="@expanded_libdir@/@PACKAGE@/libnjbglue.so" />
+	<dllmap dll="libnjb" target="libnjb.so.5" />
+	<dllmap dll="libnjbglue" target="@expanded_libdir@/@PACKAGE@/libnjbglue.so" />
 </configuration>
 
